engine/core/src/main/java/org/datacleaner/util/convert/ResourceConverter.java

```diff
@@ -94,9 +104,24 @@ public String getScheme() {
 
     /**
      * Constructs a {@link ResourceConverter} using a default set of handlers
+     * 
+     * @deprecated use another constructor instead
```

Comment:
Well there are 4 other constructors. It seems needless to mention them all - the message is to pick whatever constructor is feasible, just not this one :-)